manifests/00-cluster-role.yaml

```diff
@@ -59,6 +60,14 @@ rules:
   - watch
   - update
 
+- apiGroups:
+  - rbac.authorization.k8s.io
+  resources:
+  - roles
+  verbs:
+  - escalate
+  - bind
```

Comment:
See https://kubernetes.io/docs/reference/access-authn-authz/rbac/#privilege-escalation-prevention-and-bootstrapping